docs/pt/Building-Alluxio-Master-Branch.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: global
+title: Construindo o Alluxio Master Branch
+nickname: Construindo o Master Branch
+group: Resources
+---
+
+* Table of Contents
+{:toc}
+
+Este guia descreve os passos para compilar o Alluxio do início.
+
+O pré-requisito para este guia é que você tenha [Java 7 (or above)](Java-Setup.html),
```

Comment:
It looks like this and many other links are broken since the docs haven't been added to the Portuguese documentation yet. For now, could you add `../en` to the start of links (e.g. ../en/Java-Setup.html) so that the links go to the English docs?